metricbeat/module/traefik/health/_meta/fields.yml

```diff
@@ -0,0 +1,45 @@
+- name: health
+  type: group
+  description: >
+    Metrics obtained from Traefik's health API endpoint
+  fields:
+    - name: uptime
```

Comment:
One trick you can do here not to have to specify the group and describe is to directly write `uptime.sec`. The generators will do the rest of the magic for you.